README.md

```diff
@@ -1 +1,50 @@
-## TO-DO Readme
+# Navigator App
+
+This application allows users to know what is the shortest and the second shortest ways from a station to another navigating by car using the Floyd-Marshall Algorithm. Also it is possible to select bus as mean of transport, know what bus the user has to get on and where to change of bus.
+The map of stations is the following:
+![image](https://github.com/nazabucciarelli/navigator-project-solvd/assets/84046180/8b34b14c-20c3-41fe-9e56-6eb1f24f3fb8)
+
+
+## Explanation
+
+In order to start doing this project, we firstly did a design plan representing all the entities and responsibilities of each one, so all the team members can understand what we are doing. Then, we started with the development of the project itself. We applied the MVC pattern and used very good programming practices.
+
+## Technologies
+
+- Java
+- MySQL
+- Maven
+- Git
+
+## Set-Up
+
+To run this project you will need an updated version of Java.
```

Comment:
maybe here we can specify explicitly to have installed jdk 21

Reply:
yes can be, I had jdk 20, which isn't old, and it didn't work.

Reply:
Good observation, I've already modified it